tests/pipeline/test_extraction.py

```diff
@@ -147,7 +147,7 @@ def test_sextractor_extract_once( decam_datastore, extractor ):
         extractor.pars.test_parameter = uuid.uuid4().hex
         sourcelist, sourcefile = run_sextractor(decam_datastore.image, extractor)
 
-        assert sourcelist.num_sources == 5611
+        assert sourcelist.num_sources == 5623
```

Comment:
I suspect these changes (and the ones below) came in because you set the RESCALE_WEIGHTS parmeter in sextractor to Y instead of N.  Unless you know why we want to use that, we should keep that at N, which would mean reverting these changes.  (That will probably also allow us to put the tolerances in test_astro_cal.py back to 0.01/3600. degrees instead of 0.05/3600.  In any event, I think we want to leave that tolerance very small, because it's checking to make sure that we got the WCS solution we expected to.)